src/connection.ts

```diff
@@ -2330,6 +2331,7 @@ const TokenBalanceResult = pick({
   accountIndex: number(),
   mint: string(),
   owner: optional(string()),
+  programId:optional(string()),
```

Comment:
I can see that you haven't run any tests or build scripts, because this line of code isn't formatted.